pyproject.toml

```diff
@@ -82,7 +85,7 @@ extend_exclude = ".tox,docs,generated,src/codegen/metadata,src/codegen/templates
 [tool.pytest.ini_options]
 addopts = "--doctest-modules --strict-markers"
 filterwarnings = ["always::ImportWarning", "always::ResourceWarning"]
-testpaths = ["tests"]
+testpaths = [ "tests" , "generated/nidaqmx/_install_daqmx.py"]
```

Comment:
Is this for doctests? If so, add all of generated/nidaqmx/, not just your new module.

Reply:
getting below error when I try to add generated/nidaqmx/

![image](https://github.com/ni/nidaqmx-python/assets/927562/60c7bcfa-8bb5-4ff0-a4d1-284b02cd04bf)

Reply:
Yes, that doctest is formatted wrong.

Reply:
@bkeryan , When I try to enable doctests for generated/nidaqmx/ I observed 5 failures and I may not be the right person to fix those issues. 
I have created a [work item](https://ni.visualstudio.com/DefaultCollection/DevCentral/_workitems/edit/2694479) to track and enable doctests in the next iteration and for now I have disabled doctests.